src/Pelagos/Util/DOIutil.php

```diff
@@ -47,6 +54,7 @@ public function __construct()
         $this->doishoulder = $parameters['doi_api_shoulder'];
         $this->doiusername = $parameters['doi_api_user_name'];
         $this->doipassword = $parameters['doi_api_password'];
+        $this->url = $parameters['url'];
```

Comment:
perhaps something more descriptive, such as doi_api_url as our parameters files have a lot in them